src/domain/datasets/services/src/dataset_env_var_service_static_impl.rs

```diff
@@ -0,0 +1,93 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use database_common::DatabasePaginationOpts;
+use dill::*;
+use kamu_core::InternalError;
+use kamu_datasets::{
+    DatasetEnvVar,
+    DatasetEnvVarListing,
+    DatasetEnvVarService,
+    DatasetEnvVarValue,
+    DeleteDatasetEnvVarError,
+    GetDatasetEnvVarError,
+    ModifyDatasetEnvVarError,
+    SaveDatasetEnvVarError,
+};
+use opendatafabric::DatasetID;
+use uuid::Uuid;
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+pub struct DatasetEnvVarServiceStaticImpl {}
```

Comment:
Gotta say that I really dislike the name `Static` - it doesn't tell me what this implementation actually does.

You already have `DatasetKeyValueServiceStaticImpl` and this is a real implementation that fetches variables from system env vars.

In case of this service - it looks more like `Null` implementation, so I would advise naming it `DatasetEnvVarServiceNull` and consider renaming `DatasetKeyValueServiceStaticImpl` to `DatasetKeyValueServiceSysEnv`.

Reply:
Note that we use suffix `Impl` only when there is exactly one implementation of a service, e.g. `IngestServiceImpl`. When there are multiple we add only the variety suffix without `Impl` e.g. `ObjectRepositoryS3`. 

It's a nitpick, but it's nice to keep the code consistent.